.github/workflows/qa-deploy.yml

```diff
@@ -197,150 +198,21 @@ jobs:
           echo "::add-mask::$webapp_name"
           echo "::set-output name=webapp-name::$webapp_name"
 
+      - name: Stop App Service
+        if: false
+        env:
+          AZURE_RESOURCE_GROUP: "bw-qa-env"
+        run: az webapp stop --name ${{ steps.retrieve-secrets.outputs.webapp-name }} --resource-group $AZURE_RESOURCE_GROUP
+
```

Comment:
Should we remove this now?

Reply:
Yeah, probably. I'll put in another PR, but there is no rush on that one.